icu4j/main/classes/core/src/com/ibm/icu/impl/units/MeasureUnitImpl.java

```diff
@@ -0,0 +1,756 @@
+// © 2020 and later: Unicode, Inc. and others.
```

Comment:
As Hugo said, once #1275 is checked in, make sure MeasureUnitImpl.java and related files are removed from this PR (or is updated based on what you need).